src/Tests.props

```diff
@@ -2,7 +2,7 @@
   <Import Project="$(MSBuildThisFileDirectory)Tests.local.props" Condition="Exists('$(MSBuildThisFileDirectory)Tests.local.props')" />
 
   <PropertyGroup>
-    <NoWarn>$(NoWarn);1591;IDE1006;CS0618</NoWarn>
+    <NoWarn>$(NoWarn);CS0618</NoWarn>
```

Comment:
1591 (xml comments) moved globally for non-packable projects
IDE1006 was naming violations; there were none
CS0618 is obsolete warnings; seems appropriate to leave here for test projects